components/ShareLink/ShareLink.tsx

```diff
@@ -0,0 +1,49 @@
+import { ShareIcon } from 'components/Icons/ShareIcon';
+import styles from './ShareLink.module.scss';
+
+type ShareLinkProps = {
+    url: string;
+    title: string;
+    description?: string;
+    linkText?: string;
+};
+
+export const ShareLink = ({
+    url,
+    title,
+    description,
+    linkText = 'Share',
+}: ShareLinkProps) => {
+    const handleKeydown = (
+        event: React.KeyboardEvent<HTMLParagraphElement>,
+    ) => {
+        if (event.key === 'Enter') {
+            openShareMenu();
+        }
+    };
+
+    const openShareMenu = () => {
+        if (navigator.share) {
+            navigator.share({
+                title: title,
+                url: url,
+                text: description,
+            });
+        } else {
+            window.location.href = 'mailto:?subject' + title;
+        }
+    };
+
+    return (
+        <div
```

Comment:
**Question**: Should this be a div element? wouldnt an anchor element be more appropriate.

Reply:
I've used a div here as it displays as a block and the element doesn't have a href attribute. In general, I think an anchor might be more accessible? I would remove the styling and re-apply it to the sharelink child. I'm not sure if this makes a difference though, I will ask in the accessibility channel

Reply:
I've checked and this should be a button, I've updated the styles accordingly.